### ISSUE TITLE
Decoy rate based on installs will be to low

### ISSUE THREAD

**Issue by jorants** [minvws/nl-covid19-notification-app-coordination#70]
### Describe the bug, issue or concern

The following seems to be the calculation for the decoy rate:
 rate = (k * 20)/I
where k = number of key uploads, I= number of installs.

However, k is a measure of actual usage while I is not. Since I overestimates the actual usage the rate will end up to low. 

This is easily fixed, to see this, base k on the expected number of uploads:

 k = t*(I/P)

where t = positive tests, and P = whole population= 17M. 
Substituting back we get
 
rate = (t*(I/P)*20)/I = (t/P)*20

Or simply: make the rate equal to the positive test rate times 20.

### Governance
- [x] I've read the contributing document https://github.com/minvws/.github/blob/master/CONTRIBUTING.md
- [x] I've read and understand the Code of Conduct https://github.com/minvws/.github/blob/master/CODE_OF_CONDUCT.md
- [x] I understand that any contributions or suggestions I made may make it into the actual code. I've read the License https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/LICENSE.txt and the contributor license agreement https://github.com/minvws/nl-covid19-notification-app-coordination/blob/master/CLA.md

**Comment by damaarten** [comment]
Thanks a lot for reviewing the specification and for this feedback!
The 1 to 20 rate is a ball-park ratio we agreed upon with the operational support team early in the project.
Most important about decoy traffic is that it should be indistinguishable from real uploads.
The rate is less important. If the rate were to be near to 1, that would make any upload traffic more likely to be a real upload. But this is not the case.
We can however adjust the decoy probability for all apps, it is a server configuration value that all apps fetch from the server whenever it changes, as part of the key download process. Once the app (system) is fully operational, we will periodically evaluate if we need to adjust this value, based on the number of app installs and daily infection rates.
So in summary, we are aware that the rate of 20 to 1 may turn out different in practice, and we will adjust the value when needed.
We are flexible in adjusting.
I will discuss in the team if we want to add to the spec that we will monitor the decoyProbability value and adjust when needed.
